src/test/spec/unified_runner/operation/bulk_write.rs

```diff
@@ -31,136 +38,22 @@ impl<'de> Deserialize<'de> for WriteModel {
         #[derive(Debug, Deserialize)]
         #[serde(rename_all = "camelCase")]
         enum WriteModelHelper {
```

Comment:
Is this impl still needed?  This looks equivalent to what I think `#[derive(Deserialize)]` on `WriteModel` would produce now.

Reply:
[this limitation](https://github.com/mongodb/mongo-rust-driver/pull/1034#discussion_r1499779896) still holds unfortunately, the test files use a "tagged" representation whereas the `ops` array doesn't